app/controllers/admin/auth/passwords_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Admin::Auth::PasswordsController < ::Devise::PasswordsController
+class Admin::Auth::PasswordsController < Admin::AdminController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.